src/Uno.Extensions.Navigation.Toolkit/ServiceCollectionExtensions.cs

```diff
@@ -35,6 +68,7 @@ public static class ServiceCollectionExtensions
 	/// <param name="initialViewModel">[optional] Initial navigation viewmodel</param>
 	/// <param name="initialNavigate">[optional] Callback to drive initial navigation for app</param>
 	/// <returns>The created IHost</returns>
+	[Obsolete("Use the IApplicationBuilder", true)]
```

Comment:
Remove - We still need to support this in case someone wants to host navigation elsewhere in the application